src/api/app/assets/stylesheets/webui2/project-package-show-grid.scss

```diff
@@ -1,28 +1,28 @@
-.package-source-files {
-  grid-area: source-files;
+.package-source-files, .project-packages {
+  grid-area: list;
 }
 
-.package-build-results {
+.package-build-results, .project-build-results {
   grid-area: build-results;
 }
 
-.package-comments {
+.package-comments, .project-comments {
   grid-area: comments;
 }
 
-.package-show {
+.package-show, .project-show {
```

Comment:
I am nitpicking, but it seems there is no need to distinguish between package and project related content.
Let's call it `.overview` here and `comments`, `build-results`, `files` above (or `list`, `file-list`, ...).

Reply:
Good idea.